packages/augur-ui/src/modules/markets-list/components/landing-hero.styles.less

```diff
@@ -63,19 +63,18 @@
   }
 
   @media @breakpoint-mobile {
-    height: 245px;
+    height: 100%;
     padding: @size-24 @size-16;
 
     > img {
-      height: 245px;
-      max-height: 245px;
+      height: 100%;
     }
 
     > div:first-of-type {
       align-items: center;
       justify-content: center;
       max-width: 340px;
-      margin: auto;
```

Comment:
need to have content be centered
<img width="791" alt="Screen Shot 2019-12-30 at 1 48 58 PM" src="https://user-images.githubusercontent.com/6775839/71602057-25617500-2b0b-11ea-95f7-4c98ba071432.png">